freetextresponse/freetextresponse.py

```diff
@@ -149,6 +164,10 @@ def workbench_scenarios():
         default='',
         scope=Scope.user_state,
     )
+    student_answer_type = String(
```

Comment:
Sure, I thought about doing that way but nothing else was using that method.  Fixed